frontend/components/ui/footer.tsx

```
@@ -0,0 +1,237 @@
+                            </p>
+                            <div className='mt-4 font-semibold'>
+                                <h4>463 7th Ave, NY 10018, USA</h4>
+                                <h4 className='mt-1.5'>+123 88 9900 456</h4>
+                            </div>
+                        </div>
+                    </div>
+                    
+                    <div className='mb-6 md:ml-16 lg:mb-0'>
+                        <h2 className='relative pb-4 mb-5 text-2xl font-semibold text-white'>
+                            Useful Links
+                            <span className='absolute bottom-0 left-0 w-10 h-1 rounded-full bg-theme-primary'></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)